rust/xaynet-client/src/mobile_client/mod.rs

```diff
@@ -49,6 +54,8 @@ impl MobileClient {
     /// Fails if the crypto module cannot be initialized.
     pub fn init(
         url: &str,
+        der_certificates: Option<Vec<PathBuf>>,
```

Comment:
Maybe we should move them at the end of the parameters list because both are optional

Reply:
agree.